src/xGadgetron/cGadgetron/tests/mrtests.cpp

```diff
@@ -641,12 +642,15 @@ int main ( int argc, char* argv[])
 		
         std::string SIRF_PATH;
         if (argc==1)
-            SIRF_PATH = sirf::getenv("SIRF_PATH", true);
+        {
+            std::stringstream sssirfpath;
+            sssirfpath << sirf::getenv("SIRF_INSTALL_PATH", true) << "/share/SIRF-" << SIRF_VERSION_MAJOR << "." << SIRF_VERSION_MINOR;
+            SIRF_PATH = sssirfpath.str();
```

Comment:
I find this all a bit fragile. why are we not simply passing the filenames to the test executable?

Reply:
The MR tests require multiple different input files depending on what trajectory is looked at.

I could split the tests into multiple executables, `MR_TESTS_RPE`, `MR_TESTS_CARTESIAN` etc. then we could pass the filename directly to the test in the CMake command
```
ADD_TEST(NAME MR_TESTS_CPLUSPLUS
         COMMAND MR_TESTS_CPLUSPLUS + (here goes the filename, I guess)
         WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
```  
Would also solve the windows path concatenation problem, and also the version in C++ would not be required...

Reply:
sounds good, but you can of course also pass 2 filenames to the executable, so not sure if it's required. (I assume you have some code to check if the RPE is enabled or not anyway)

Reply:
Yes that check is done. I guess I will pass two filenames for now that should be quickest.

Reply:
done